known-issues.md

```diff
@@ -259,28 +259,39 @@ Note: There is a known issue with Visual Studio 2015 Update 3 RC, as a result of
 
 ### Missing SDK
```

Comment:
Should we move this section to the top of the known issues page (ie. under latest release) ?